.github/workflows/gh-pages.yml

```diff
@@ -10,6 +10,7 @@ on:
   # Runs every time a PR is open against master
   pull_request:
     branches: [ master ]
+
   workflow_dispatch:
 
 jobs:
```

Comment:
Can we rename `emaxx-docs` to `eamxx-docs`?

Reply:
Could not comment on the actual line, since it was not a touched one, but it's the job name

Reply:
For a second there, I pictured Emaxx: the C++/Kokkos port of GNU Emacs. <shudder>